### PR TITLE
Added ifexists parameter to Cluster Loader content

### PR DESCRIPTION
**Pull request by ahardin-rh**
@sjug PTAL :balloon:

### CODE REVIEW

**Comment by sjug** [comment]
/lgtm

Thanks @ahardin-rh!

**Comment by ahardin-rh** [comment]
@openshift/team-documentation Please peer review

File: scaling_performance/using_cluster_loader.adoc [hunk 1]
```diff
@@ -110,6 +110,11 @@ identical namespaces will be appended to `Basename` to prevent collisions.
 |A string. One definition of what tuning set you want to apply to the objects,
 which you deploy inside this namespace.
 
+|`ifexists`
+|A string containing either `reuse` or `delete`. Defines what the tool will do if
```
Comment (kalexand-rh) [inline scaling_performance/using_cluster_loader.adoc]: s/will do/does

File: scaling_performance/using_cluster_loader.adoc [hunk 2]
```diff
@@ -110,6 +110,11 @@ identical namespaces will be appended to `Basename` to prevent collisions.
...
+it finds a project or namespace that has the same name of the project or
+namespace it will create during execution.
```
Comment (kalexand-rh) [inline scaling_performance/using_cluster_loader.adoc]: s/will create/creates

**Review by kalexand-rh — COMMENTED** [review]
I'd drop the future tense, but LGTM

**Pull request merged**